### DISCUSSION TITLE
ZenBleed (CVE-2023-20593)

### DISCUSSION THREAD

**Discussion by vielmetti** [siderolabs/talos#7529] [General]
Please describe how Talos Linux is responding to the ZenBleed (CVE-2023-20593) issue, which affects systems AMD Zen2 chips. 

Mitigation is via updated linux-firmware, or if the firmware is unavailable, via a "chicken bit" that disables the affected feature. Updated kernels are available.

I did a search as best I could for information in this repo, and didn't come up with anything; happy to be pointed at existing docs.

**Comment by frezbo** [comment]
This will be fixed by shipping Linux kernel 6.1.41 (which sets the chicken bit, since there is no firmware microcode fixed for all AMD CPU's) to be shipped with Talos 1.4.7 (should be today or tomorrow)

**Reply by smira** [reply]
https://github.com/siderolabs/talos/releases/tag/v1.4.7 contains a Linux upstream workaround ("chicken bit")

See [Zenbleed description](https://lock.cmpxchg8b.com/zenbleed.html) for more details

**Reply by frezbo** [reply]
https://github.com/siderolabs/talos/security/advisories/GHSA-4ffv-x7jf-cv68
